### PR TITLE
fix for non-functional (un)publish dates on the message form

### PR DESCRIPTION
**Pull request by mollux**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

The `Publish at (date/time)` and `Unpublish at (date/time)` fields are not working on the messages form.
This is because the original attributes (defining the `data-type` and `class`) are not present as an extra attribute is added in the form.
Screenshot 2023-09-16 at 08 28 57

This PR fixes this by adding the original attributes.

Screenshot 2023-09-16 at 08 29 56

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. create a marketing message and verify you can select a date / time in both fields.

### CODE REVIEW

**Review by kuzmany — DISMISSED** [review]
👍

File: app/bundles/CoreBundle/Form/Type/AbstractFormStandardType.php [hunk 1]
```diff
@@ -111,8 +111,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             );
 
             if (!$builder->has('publishUp') && method_exists($options['data'], 'getPublishUp')) {
-                $builder->add('publishUp', PublishUpDateType::class, ['attr' => ['readonly' => $readonly]]);
-                $builder->add('publishDown', PublishDownDateType::class, ['attr' => ['readonly' => $readonly]]);
+                $builder_options = [
```
Comment (kuzmany) [inline app/bundles/CoreBundle/Form/Type/AbstractFormStandardType.php]: The preferred style for variable names is camelCase. So, based on that, $builderOptions would be more in line with those guidelines.

File: app/bundles/CoreBundle/Form/Type/AbstractFormStandardType.php [hunk 2]
```diff
@@ -111,8 +111,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
+                  'attr' => [
+                      'class'       => 'form-control',
+                      'data-toggle' => 'datetime',
+                      'readonly'    => $readonly,
+                    ],
+                ];
+                $builder->add('publishUp', PublishUpDateType::class, $builder_options);
+                $builder->add('publishDown', PublishDownDateType::class, $builder_options);
```
Comment (escopecz) [inline app/bundles/CoreBundle/Form/Type/AbstractFormStandardType.php]:
```suggestion
                $builderOptions = [
                  'attr' => [
                      'class'       => 'form-control',
                      'data-toggle' => 'datetime',
                      'readonly'    => $readonly,
                    ],
                ];
                $builder->add('publishUp', PublishUpDateType::class, $builderOptions);
                $builder->add('publishDown', PublishDownDateType::class, $builderOptions);
```

**Review by escopecz — APPROVED** [review]
I was able to reproduce the issue and it is fixed as the screenshots in the PR description shows. Thank you! 👍

**Pull request merged**